packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/index.vue

```diff
@@ -169,10 +167,7 @@ const onDecode = async (scannedCodeValue: string) => {
                         :data-testid="`nc-form-input-cell-${field.label || field.title}`"
                         :class="`nc-form-input-${field.title?.replaceAll(' ', '')}`"
                         :column="field"
-                        :edit-enabled="editEnabled[index]"
-                        @click="editEnabled[index] = true"
-                        @cancel="editEnabled[index] = false"
-                        @update:edit-enabled="editEnabled[index] = $event"
+                        :edit-enabled="true"
```

Comment:
What is the reason changing that?

Reply:
It makes the Cell component to render the input element by default. Removes unwanted compute.

Reply:
if we don't need these in all types of forms, you may also update `Form.vue` and `packages/nc-gui/pages/index/[typeOrId]/form/[viewId]/index/survey.vue`.

Reply:
survey.vue not yet updated

Reply:
https://github.com/nocodb/nocodb/blob/d575c87316529c8a13e86669fe717b1b5cd93154/packages/nc-gui/pages/index/%5BtypeOrId%5D/form/%5BviewId%5D/index/survey.vue#L303-L306

Reply:
Updated